docs/configuration.md

```diff
@@ -140,15 +144,16 @@ The following subsections cover all CLI flags which promscale supports. You can
-| startup.upgrade-extensions            | boolean |     true      | Upgrades TimescaleDB & Promscale extensions.                                                                                                                                                                                     |
-| startup.upgrade-prerelease-extensions | boolean |     false     | Upgrades to pre-release TimescaleDB, Promscale extensions.                                                                                                                                                                       |
-| startup.use-schema-version-lease      | boolean |     true      | Use schema version lease to prevent race conditions during migration.                                                                                                                                                            |
+| Flag                                  | Type    | Default       | Description                                                                                                                                                                                                                                                                                                                                                                                         |
+|---------------------------------------|:-------:|:-------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| startup.dataset.config                | string  | "" (disabled) | **DEPRECATION NOTICE**: *this flag is going to be removed in a future version, please follow the [upgrade guide](dataset.md#upgrading-from-startupdatasetconfig)*. Dataset configuration in YAML format for Promscale. It is used for setting various dataset configuration like default metric chunk interval. For more information, please consult the following resources: [dataset](dataset.md) |
+| startup.dataset *(config.yaml only)*  |  yaml   | "" (disabled) | Dataset configuration in YAML format for Promscale. It is used for setting various dataset configuration like default metric chunk interval. For more information, please consult the following resources: [dataset](dataset.md)                                                                                                                                                                    |
```

Comment:
Isn't the type string? This should be set to path to YAML config file, right?

Reply:
No, it's a yaml mapping node. That's why it's only supported in the config.yaml file.

```
startup.dataset:
    metrics:
      default_chunk_interval: 8h  
```